assets/css/main.css

```diff
@@ -0,0 +1,106 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Lato:wght@100&family=Open+Sans:wght@300&family=Poppins&family=Roboto:ital,wght@0,700;1,100&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Arial", "Helvetica", sans-serif;
+  font-size: 12px;
+}
+
+/* variables */
+:root {
+  --dark-grey: #3c3a39;
+  --black: #1c1a19;
+  --main-white: #fff;
+}
+
+/* header section */
+#header {
+  width: 100%;
+  background-color: var(--dark-grey);
+  height: 92px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 20px;
+}
+
+#header a {
+  text-decoration: none;
+  color: var(--main-white);
+  font-family: "Roboto", "Helvetica", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.nav_icon {
+  cursor: pointer;
+}
+
+/* end of header
+section */
+
+/* headline section */
+#headline {
+  height: 590px;
+  width: 100%;
+  background-color: var(--black);
+  padding: 24px;
+  position: relative;
+  background-image: url('../images/Illustration.png');
```

Comment:
- [ ] Great job adding the background image the right way but I can see that this image is overlapping the `headline` section text so please fix that.

![image](https://user-images.githubusercontent.com/68796232/158793237-b4913cb2-3ab3-47a1-b03d-41fec4752ca7.png)
![image](https://user-images.githubusercontent.com/68796232/158793057-817f43e5-33be-4a17-b316-2c5e7a6bb002.png)

Reply:
![mobile_view](https://user-images.githubusercontent.com/98595024/158815636-64c0f395-944f-46d4-a06c-fe291789bf48.PNG)
This looks much better